packages/generate-design-tokens/css-generator.js

```diff
@@ -0,0 +1,357 @@
+        });
+        return `linear-gradient(${gradientDescription.angle}deg, ${stops.join(', ')})`;
+    };
+
+    const buildCssColor = (colorDescription) => {
+        if (colorDescription.type === 'linear-gradient') {
+            return buildCssGradient(colorDescription.value);
+        }
+
+        if (colorDescription.value.startsWith('{palette.') && colorDescription.value.endsWith('}')) {
+            const paletteColorName = colorDescription.value.replace('{palette.', '').replace('}', '');
+            usedPaleteColors.add(paletteColorName);
+            return `var(${buildPaletteColorVarName(paletteColorName)})`;
+        }
+
+        // https://regexper.com/#%2Frgba%5C%28%5C%7Bpalette%5C.%28.%2B%29%5C%7D%2C%20%28%5B01%5D%28%3F%3A%5C.%5Cd%2B%29%3F%29%5C%29%2F
+        const colorWithAlphaRegExp = /rgba\(\{palette\.(.+)\}, ([01](?:\.\d+)?)\)/;
+        const colorWithAlphaMatches = colorDescription.value.match(colorWithAlphaRegExp);
+
+        if (colorWithAlphaMatches) {
+            const colorName = colorWithAlphaMatches[1];
+            const alpha = colorWithAlphaMatches[2];
+            usedRawPaletteColors.add(colorName);
+            return `rgba(var(${buildRawPaletteColorVarName(colorName)}), ${alpha})`;
+        }
+
+        throw new Error(`Unknown color format: ${colorDescription.value}`);
+    };
+
+    const lightColors = Object.entries(designTokens.light)
+        .map(
+            ([colorName, colorDescription]) =>
+                `${buildColorVarName(colorName)}: ${buildCssColor(colorDescription)};`
+        )
+        .join('\n');
+
+    const darkModeColors = Object.entries(designTokens.dark)
+        .map(
+            ([colorName, colorDescription]) =>
+                `${buildColorVarName(colorName)}: ${buildCssColor(colorDescription)};`
+        )
+        .join('\n');
+
+    const forceLightModeColors = Object.keys(designTokens.dark)
+        .map(
+            (colorName) => `${buildColorVarName(colorName)}: ${buildCssColor(designTokens.light[colorName])};`
+        )
+        .join('\n');
+
+    const paletteColors = Array.from(usedPaleteColors)
+        .map((colorName) => {
+            return `${buildPaletteColorVarName(colorName)}: ${designTokens.global.palette[colorName].value};`;
+        })
+        .join('\n');
+
+    const rawPaletteColors = Array.from(usedRawPaletteColors)
+        .map((colorName) => {
+            const colorValue = designTokens.global.palette[colorName].value;
+            const [r, g, b] = fromHexToRgb(colorValue);
+            return `${buildRawPaletteColorVarName(colorName)}: ${r}, ${g}, ${b};`;
+        })
+        .join('\n');
+
+    const borderRadii = Object.entries(designTokens.radius)
+        .map(
+            ([radiusName, radiusDescription]) =>
+                `${buildBorderRadiusVarName(radiusName)}: ${buildRadius(radiusDescription)};`
+        )
+        .join('\n');
+
+    const textPresets = {
+        1: {
```

Comment:
I've extracted these values from `text.tsx`, but I think these tokens should be in https://github.com/Telefonica/mistica-design/blob/production/tokens/movistar.json#L1384

Reply:
Also, reviewing the values there's something weird, text4 and text5 have the same size and line height, and it seems that it starts in text 3, that should be size: 16px mobile and 18 desktop

Reply:
https://github.com/Telefonica/mistica-design/pull/1817